resources/public/api/conf/1.0/checkpoint_create.yaml

```diff
@@ -0,0 +1,49 @@
+summary: Create a Checkpoint
+description: |
+  This endpoint allows a developer to set up a checkpoint for the stateful data in the sandbox environment for a particular NINO. 
```

Comment:
Do we need to make this description plural, i.e `checkpoints`? Or can we leave it as is?

Reply:
Atm developers can only make one checkpoint per Nino, so I think it makes more sense to keep it singular

Reply:
Thanks!